### DISCUSSION TITLE
v0.64.0

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#3571] [Announcements]
The `ALLOWED_DOMAINS` environment variable was moved into the UI under **Settings** -> **Security**. Upon upgrading, any existing values set in the environment will be automatically migrated and you can safely remove the `ALLOWED_DOMAINS` value post-upgrade.

We also introduced stricter validation of all environment variables to help with debugging and setup of the community edition, depending on your environment this may result in new errors or warnings being output upon server startup.

### Enhancements

* Migration of `ALLOWED_DOMAINS` to a team level setting (see above) by @coreyja in https://github.com/outline/outline/pull/3489
* New TLDraw, Otter.ai, Gliffy, JSFiddle, and Scribe embed integrations.
* Document can now be fetched through API with knowing the slug by @coreyja in https://github.com/outline/outline/pull/3453
* Multiple performance improvements for large documents with collaborative editing enabled https://github.com/outline/outline/pull/3567
* Improved performance by requesting less db columns when calculating collection permissions in https://github.com/outline/outline/pull/3498
* Improved performance for large collections by refactoring sorting in https://github.com/outline/outline/pull/3475
* ioredis is now configurable via environment variables by @heilerich in https://github.com/outline/outline/pull/3365
* Disabled state of embeds now persists per-document https://github.com/outline/outline/pull/3407
* Suspended users are no longer shown in the document facepile in https://github.com/outline/outline/pull/3497
* Added initial support to import data from Notion (**Settings** -> **Import** -> **Notion**)
* Publicly shared links now include title and description in metadata for improved previews in third party services
* Improve speed of Azure login (parallelize two slow API requests) in https://github.com/outline/outline/commit/bb074edb0d464c59dfb4a43c5da1707addd90e45
* Turkish language translations were added 🇹🇷 

#### Editor
* Emojis and embeds can now be copied to plain text clipboard in https://github.com/outline/outline/pull/3561
* Floating toolbar now only appears on mouse up when selecting text with the mouse in https://github.com/outline/outline/pull/3561
* Links to anchors in other pages are no longer broken when the page is renamed in https://github.com/outline/outline/commit/0577c73f0661b75aafea6a768b4a652781c81f69
* Applying a template no longer overwrites an existing title in https://github.com/outline/outline/commit/7fa0199dcabf285d0b4a317976060d0824946911
* "Referenced by" is now "Backlinks"

#### Platform
* A shared persistence debounce between docs could result in some docs not saving on the server in high-concurrency editing environments https://github.com/outline/outline/pull/3401
* Fixed translation strings throughout the UI by @Limezy in https://github.com/outline/outline/pull/3417 https://github.com/outline/outline/pull/3441
* Fixed an issue where the "last edited by" timestamp could be updated when the user had made no changes
* Fixed an issue where CRDT creation from markdown state touches document updated timestamp in https://github.com/outline/outline/pull/3482
* Paginated list history headings were not rendering when there was only one unique heading by @thenanyu in https://github.com/outline/outline/pull/3496
* Data import and export codepaths were refactored and improved https://github.com/outline/outline/pull/3434
* Confirm button is now focused by default in confirmation dialogs in https://github.com/outline/outline/commit/4c15f27bb2246da41dd5008a12c62e73087b4f5c
* Fixed extra separator in collection context menu with read-only permissions in https://github.com/outline/outline/commit/b152b9f17b8da7c7c0fd78a844aa48e660f1398e
* Fixed blank screen instead of "Not found" page in some situations in https://github.com/outline/outline/commit/40e41b26a19f4a47582acf9e71272c80abda9396
* Revisions are now created on document publish in https://github.com/outline/outline/commit/ec1bc801a4e391e7528d706d5855b464a1b27093
* Collection names with slashes no longer produce an invalid export in https://github.com/outline/outline/commit/4177031d0b187db7604f6b53f9842db21d854aec
* "Settings" navigation is no longer offered to non-admins in https://github.com/outline/outline/commit/78da5e2335eace26535f5981c0b789f8c7c4a07e

## New Contributors
* @meltyshev made their first contribution in https://github.com/outline/outline/pull/3428
* @Limezy made their first contribution in https://github.com/outline/outline/pull/3439
* @gemathus made their first contribution in https://github.com/outline/outline/pull/3447
* @coreyja made their first contribution in https://github.com/outline/outline/pull/3453
* @EmirBoyaci made their first contribution in https://github.com/outline/outline/pull/3473
* @heilerich made their first contribution in https://github.com/outline/outline/pull/3365
* @rusakovdenis made their first contribution in https://github.com/outline/outline/pull/3548

**Full Changelog**: https://github.com/outline/outline/compare/v0.63.0...v0.64.0

<hr /><em>This discussion was created from the release <a href='https://github.com/outline/outline/releases/tag/v0.64.0'>v0.64.0</a>.</em>

**Comment by almereyda** [comment]
Please also recognise the maintenance release [v0.64.1](https://github.com/outline/outline/releases/tag/v0.64.1).

**Comment by jagyas** [comment]
Hello, Can anyone please help me to understand meaning of "Referenced by" is now "Backlinks" in change log? Is it similar to Obsidian backlinks?

**Reply by tommoor** [reply]
Backlinks has been a feature in Outline for a long time, it was called "Referenced by" but was renamed in this release as more products were referring to these as backlinks and we felt it was clearer to match the language used elsewhere.

**Reply by jagyas** [reply]
Can you please help me to know how I can use this feature?

**Reply by tommoor** [reply]
Backlinks are automatically created when you link from one document to another (use `cmd+k` with the editor focused to trigger a doc search window), they appear at the bottom of the linked document.

**Reply by jagyas** [reply]
Got it. Thank you so much for your help.
